planning:
  discretization:
    r_shoulder_pan_joint    0.017453292519943295
    r_shoulder_lift_joint   0.017453292519943295
    r_upper_arm_roll_joint  0.017453292519943295
    r_elbow_flex_joint      0.017453292519943295
    r_forearm_roll_joint    0.017453292519943295
    r_wrist_flex_joint      0.017453292519943295
    r_wrist_roll_joint      0.017453292519943295
  use_xyz_snap_mprim: false
  use_rpy_snap_mprim: false
  use_xyzrpy_snap_mprim: true
  use_short_dist_mprims: true
  xyz_snap_dist_thresh: 0.01
  rpy_snap_dist_thresh: 0.01
  xyzrpy_snap_dist_thresh: 0.01
  short_dist_mprims_thresh: 0.1

occupancy_grid:
  size_x: 3.0
  size_y: 2.0
  size_z: 1.35
  res: 0.01
  origin_x: -1.0
  origin_y: -1.5
  origin_z: 0.15
  max_dist: 0.5
